src/platform/common/types.ts

```diff
@@ -350,3 +350,33 @@ export interface ICellRange {
     range: Range;
     cell_type: string;
 }
+
+// Used by platform/common/scriptPathUtils
+export const IScriptPathUtils = Symbol('IConfigurationService');
+export interface IScriptPathUtils {
+    rootDirectory: string;
+    DataFrameLoading: {
+        SysPath: string;
```

Comment:
id chance all of these to be camel. cased, and i don't think `sysPath` is used outside this class, never your can remove this

Reply:
> I'm assuming the approach is ok otherwise the casing?

Yes.